<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <rosparam command="load" file="$(find rosbot_description)config/joint_state_controller.yaml" ns="/robot_exp"/>
  <node name="joint_state_controller_spawner" pkg="controller_manager" type="spawner" output="screen" args="--namespace=/robot_exp
  joint_state_controller
  camera_velocity_controller
  --shutdown-timeout 3"/>

  <param name="robot_description" command="$(find xacro)/xacro '$(find rosbot_description)/urdf/rosbot.xacro'"/>

  <node name="rosbot_spawn" pkg="gazebo_ros" type="spawn_model" output="screen" args="-urdf -param robot_description -model rosbot" />

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
  <remap from="joint_states" to="robot_exp/joint_states"/>
  
  </node>

</launch>
